src/stores/booking.ts

```diff
@@ -0,0 +1,25 @@
+import { defineStore } from "pinia";
+import type { Seance, CompleteSeance } from "@/types/seance";
+import type { TempTicket } from "@/types/reservations";
+
+interface RootState {
+  selectedSeance: Seance;
+  selectedReservation: CompleteSeance;
+  selectedTickets: TempTicket[] | [];
```

Comment:
I don't think you need to type empty array separately

Reply:
This actually doesnt work at all without empty array. I have asked and shown this to Tomek Skuta during todays TS training and he wasn't able to help